amplipi/streams.py

```diff
@@ -782,10 +785,12 @@ def info(self) -> models.SourceInfo:
     loc = f'{src_config_folder}/currentSong'
     source = models.SourceInfo(name=self.full_name(),
                               state=self.state,
-                              img_url=self.logo,
+                              img_url='static/imgs/internet_radio.png',
```

Comment:
It's probably fine, but this icon looks like it should be for FM radio, not internet radio.

Reply:
Hmm check out the fmradio icon: https://github.com/micro-nova/AmpliPi/blob/main/web/static/imgs/fmradio.png